internal/tools/go.mod

```diff
@@ -15,3 +15,157 @@ require (
 	golang.org/x/exp v0.0.0-20200331195152-e8c3332aa8e5
 	golang.org/x/tools v0.1.5
 )
+
+require (
```

Comment:
Same, why do we have 2 `require` blocks?

Reply:
Not sure why go mod tidy created a new block, I will merge them.